client/src/app/product/[productid]/page.tsx

```
@@ -22,23 +31,47 @@ export default function Page({
   }, [category]);
 
+  useEffect(() => {
+    const fetchUserReview = async () => {
+      if (!user?.email) return;
+
+      try {
+        const { data } = await axios.get(
+          `${process.env.NEXT_PUBLIC_BASE_URL}/api/review/${user.email}?productId=${productid}`
+        );
+
+        if (data && data.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)